atd-vzd/schema/atd_txdot_person.sql

```diff
@@ -67,15 +64,18 @@ CREATE TABLE public.atd_txdot_person (
     updated_by character varying(64),
     person_id integer NOT NULL,
     is_retired boolean DEFAULT false,
-    years_of_life_lost integer DEFAULT 0 NOT NULL
+    years_of_life_lost integer GENERATED ALWAYS AS (
+CASE
+    WHEN (prsn_injry_sev_id = 4) THEN GREATEST((75 - prsn_age), 0)
+    ELSE 0
+END) STORED
```

Comment:
a lot of these changes arent mine this schema just hasnt been updated in this folder in awhile.